source/admin_manual/list_of_events.rst

```diff
@@ -2602,6 +2602,22 @@ metacache_user_clean_finished
 | error                 | Error message if the upload failed                   |
 +-----------------------+------------------------------------------------------+
 
+metacache_pull_begin
+^^^^^^^^^^^^^^^^^^^^
+metacache_pull_finished
+^^^^^^^^^^^^^^^^^^^^^^^
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| source-backend-host   | From which host metacache is being pulled            |
++-----------------------+------------------------------------------------------+
+| type                  | server or client                                     |
++-----------------------+------------------------------------------------------+
+| exit_code             | exit_code for finished metacache pull commands       |
```

Comment:
This is the same as doveadm exit code? Is it a number or text? Maybe link to doveadm_error_codes?